<?xml version="1.0"?>

 <!-- Simple example with parameters.


  -->

<launch>

    <node pkg="contact_monitor" type="contactMonitor" name="contact" >
      <param name="collisions_topic_name" value="/isColliding"/>
      <param name="robot_model_name" value="robot1"/>
      <param name="actor_model_name" value="actor1"/>
    </node>

</launch>
